src/components/modal/v2/styles/components/_instructions.scss

```
@@ -18,6 +18,8 @@
 
         @include mobile {
             padding: 0px;
+            margin-top: -11px;
+            margin-bottom: 0px;
         }
```

Instead of having this here, can we move these two styles out of the `@include mobile` media query and put it right below in its own class?

```suggestion
      &.ppc {
            @include mobile {
                margin-top: -11px;
                margin-bottom: 0px;
            }
        }
```